docs/Project-Configuration/Stepback-Bisection.md

```diff
@@ -0,0 +1,56 @@
+# Stepback Bisection
+Evergreen's stepback bisection performs stepback by continuously reducing the amount of commits needed to test by half. To enable it, go to the project settings 'general' tab and enable the 'Stepback Bisection' flag. Note that the 'Stepback' flag has to be enabled as well for bisection to take place.
```

Comment:
nit: Project Settings page "General"  tab

Reply:
"Note that this flag" also would be preferable -- ideally we shouldn't call it stepback bisection in one spot and then Stepback in  the next -- consistent language is easier to follow

Reply:
With the consistent language comment, I'm trying to communicate that there exists the 'Stepback' flag and the 'Stepback bisection' flag will do nothing if the 'Stepback' flag is false.

I'll reword to: "For this flag to work, the 'Stepback' base flag will also need to be enabled."